manager/state/raft/storage/nacl.go

```diff
@@ -0,0 +1,74 @@
+package storage
+
+import (
+	"crypto/rand"
+	"fmt"
+	"io"
+
+	"golang.org/x/crypto/nacl/secretbox"
```

Comment:
@lvh Thanks for the great explanation! Good to see more acceptance of NaCl across the board.